ep/build-arrow/src/get_arrow.sh

```diff
@@ -29,22 +29,16 @@ for arg in "$@"; do
 done
 
 function checkout_code {
-  TARGET_BUILD_COMMIT="$(git ls-remote $ARROW_REPO $ARROW_BRANCH | awk '{print $1;}')"
   ARROW_SOURCE_DIR="$CURRENT_DIR/../build/arrow_ep"
   if [ -d $ARROW_SOURCE_DIR ]; then
     echo "Arrow source folder $ARROW_SOURCE_DIR already exists..."
     cd $ARROW_SOURCE_DIR
     git init .
-    EXISTS=$(git show-ref refs/heads/build_$TARGET_BUILD_COMMIT || true)
-    if [ -z "$EXISTS" ]; then
-      git fetch $ARROW_REPO $TARGET_BUILD_COMMIT:build_$TARGET_BUILD_COMMIT
-    fi
```

Comment:
I'll help finding another solution to pull a tag

Reply:
hi @jinchengchenghh , would you like to take a look at https://github.com/zhztheplayer/gluten/commit/841b9c2114b425ec0db806d7d95262666950d4bd

this should support both pulling branches/tags

Reply:
@jinchengchenghh Since CI was already passed, let's move the discussion into https://github.com/oap-project/gluten/pull/1757. Would you like to take a look to that? Thanks.